integration/monai/examples/spleen_ct_segmentation_local/jobs/spleen_ct_segmentation_local/app/config/config_fed_server.json

```diff
@@ -52,10 +52,10 @@
           "experiment_name": "monai-spleen-experiment",
           "run_name": "monai-spleen-with-mlflow",
           "experiment_tags": {
-            "mlflow-note-content": "## **MONAI experiment with spleen bundle with MLflow**"
+            "\"mlflow.note.content\"": "## **MONAI experiment with spleen bundle with MLflow**"
```

Comment:
We should probably make this change in our other json configurations that use the MLflowReceiver. ([sag_pt_mlflow](https://github.com/NVIDIA/NVFlare/blob/main/job_templates/sag_pt_mlflow/config_fed_server.conf) and [experiment_tracking](https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/experiment-tracking/mlflow/jobs))

Reply:
let me fix this, good suggestion